# HG changeset patch
# User Rik <rik@octave.org>
# Date 1509570356 25200
#      Wed Nov 01 14:05:56 2017 -0700
# Node ID d5c38fdf77f93a307f5368c7c8655c28a7045e60
# Parent  fdd67d871a72e2b6e6441e7879e0bef5835c5315
plotyy.m: Delete unuseful third demo (bug #52302).

* plotyy.m: Delete third %!demo block.

diff --git a/scripts/plot/draw/plotyy.m b/scripts/plot/draw/plotyy.m
--- a/scripts/plot/draw/plotyy.m
+++ b/scripts/plot/draw/plotyy.m
@@ -299,22 +299,8 @@ endfunction
 %!  plotyy (x,sin(pi*x), x,10*cos(pi*x));
 %! subplot (2,2,2);
 %!  surf (peaks (25));
 %! subplot (2,2,3);
 %!  contour (peaks (25));
 %! subplot (2,2,4);
 %!  plotyy (x,10*sin(2*pi*x), x,cos(2*pi*x));
 %!  axis square;
-
-%!demo
-%! clf;
-%! hold on
-%! t = (0:0.1:9);
-%! x = sin (t);
-%! y = 5 * cos (t);
-%! [hax, h1, h2] = plotyy (t, x, t, y);
-%! [~, h3, h4] = plotyy (t+1, x, t+1, y);
-%! set ([h3, h4], "linestyle", "--");
-%! xlabel (hax(1), "xlabel");
-%! title (hax(2), 'Two plotyy graphs on same figure using "hold on"');
-%! ylabel (hax(1), "Left axis is Blue");
-%! ylabel (hax(2), "Right axis is Orange");
